projects/ad777x_iio/README.txt

```diff
@@ -0,0 +1,48 @@
+Evaluation Boards/Products Supported
+------------------------------------ 
+-Products supported: AD7770, AD7771, AD7779
+
+Overview
+--------
+This is a IIO based firmware application to evalute the AD777x device.
+This code was developed and tested on SDP-K1 controller board for mbed platform : https://os.mbed.com/platforms/SDP_K1/.
+and the NucleoL552ZEQ Board for STM32 platform: https://os.mbed.com/platforms/ST-Nucleo-L552ZE-Q/
+
+Product details: 
+https://www.analog.com/en/products/ad77770.html
```

Comment:
typo: remove extra "7". Same with line 13, 14

Reply:
The typo has been corrected now